x-pack/plugins/timelines/common/types/timeline/actions/index.ts

```diff
@@ -4,159 +4,86 @@
-
-export interface CustomBulkAction {
-  key: string;
-  label: string;
-  disableOnQuery?: boolean;
-  disabledLabel?: string;
-  onClick: (items?: TimelineItem[]) => void;
-  ['data-test-subj']?: string;
-}
-
-export type CustomBulkActionProp = Omit<CustomBulkAction, 'onClick'> & {
-  onClick: (eventIds: string[]) => void;
-};
-
-export interface BulkActionsProps {
-  eventIds: string[];
-  currentStatus?: AlertStatus;
-  query?: string;
-  indexName: string;
-  setEventsLoading: SetEventsLoading;
-  setEventsDeleted: SetEventsDeleted;
-  showAlertStatusActions?: boolean;
-  onUpdateSuccess?: OnUpdateAlertStatusSuccess;
-  onUpdateFailure?: OnUpdateAlertStatusError;
-  customBulkActions?: CustomBulkActionProp[];
-}
+/** The default minimum width of a column (when a width for the column type is not specified) */
+export const DEFAULT_COLUMN_MIN_WIDTH = 180; // px
 
-export interface HeaderActionProps {
-  width: number;
-  browserFields: BrowserFields;
-  columnHeaders: ColumnHeaderOptions[];
-  fieldBrowserOptions?: FieldBrowserOptions;
-  isEventViewer?: boolean;
-  isSelectAllChecked: boolean;
-  onSelectAll: ({ isSelected }: { isSelected: boolean }) => void;
-  showEventsSelect: boolean;
-  showSelectAllCheckbox: boolean;
-  sort: SortColumnTable[];
-  tabType: string;
-  timelineId: string;
-}
-
-export type GenericActionRowCellRenderProps = Pick<
-  EuiDataGridCellValueElementProps,
-  'rowIndex' | 'columnId'
->;
-
-export type HeaderCellRender = ComponentType | ComponentType<HeaderActionProps>;
-export type RowCellRender =
-  | JSXElementConstructor<GenericActionRowCellRenderProps>
-  | ((props: GenericActionRowCellRenderProps) => JSX.Element)
-  | JSXElementConstructor<ActionProps>
-  | ((props: ActionProps) => JSX.Element);
-
-interface AdditionalControlColumnProps {
-  ariaRowindex: number;
-  actionsColumnWidth: number;
-  columnValues: string;
-  checked: boolean;
-  onRowSelected: OnRowSelected;
-  eventId: string;
-  id: string;
-  columnId: string;
-  loadingEventIds: Readonly<string[]>;
-  onEventDetailsPanelOpened: () => void;
-  showCheckboxes: boolean;
-  // Override these type definitions to support either a generic custom component or the one used in security_solution today.
-  headerCellRender: HeaderCellRender;
-  rowCellRender: RowCellRender;
-}
+export type ColumnHeaderType = 'not-filtered' | 'text-filter';
 
-export type ControlColumnProps = Omit<
-  EuiDataGridControlColumn,
-  keyof AdditionalControlColumnProps
-> &
-  Partial<AdditionalControlColumnProps>;
-export interface BulkActionsObjectProp {
-  alertStatusActions?: boolean;
-  onAlertStatusActionSuccess?: OnUpdateAlertStatusSuccess;
-  onAlertStatusActionFailure?: OnUpdateAlertStatusError;
-  customBulkActions?: CustomBulkAction[];
-}
-export type BulkActionsProp = boolean | BulkActionsObjectProp;
-
-export type AlertWorkflowStatus = 'open' | 'closed' | 'acknowledged';
+export const defaultColumnHeaderType: ColumnHeaderType = 'not-filtered';
```

Comment:
Nit: Same here, unless this is actually being used as a type?

Reply:
Yeah, this is used as a type on line 77 in this file.